website/docs/docs/cloud-integrations/semantic-layer/power-bi.md

```diff
@@ -0,0 +1,154 @@
+   <Lightbox src="/img/docs/cloud-integrations/sl-pbi/pbi-sidepanel.jpg" title="Select tables in the side panel" />
+
+Now that you've configured the connector, you can configure published reports in the next section to use the connector.
+
+## Configure published reports
+
+After publishing a report and the first time you hit **Publish** on a given report, configure Power BI Service to use your organization’s On-premises Data Gateway to access data from the dbt Semantic Layer:
+
+1. On the top right, click on **Settings > Power BI settings**.
+   <Lightbox src="/img/docs/cloud-integrations/sl-pbi/pbi-settings.jpg" title="Navigate to Settings > Power BI Settings" />
+2. Navigate to the **Semantic models** tab and select your report on the sidebar on the left.
+3. Under **Gateway and cloud connections**, select the **On-premises Data Gateway** where your IT admin has installed the dbt Semantic Layer connector. 
+   - If the Status is **Not configured correctly**, you’ll have to configure it.
+   <Lightbox src="/img/docs/cloud-integrations/sl-pbi/pbi-gateway-cloud-connections.jpg" title="Configure the gateway connection" />
+4. Click on the arrow under **Actions** and then, click on **Manually add to gateway**.
+   <Lightbox src="/img/docs/cloud-integrations/sl-pbi/pbi-manual-gateway.jpg" title="Manually add to gateway" />
+5. Provide a name for your connection and enter your connection details. 
+   - Set the connection as **Encrypted** (Required). Failing to do so will result in the Semantic Layer servers rejecting the connection.
+   <Lightbox src="/img/docs/cloud-integrations/sl-pbi/pbi-encrypted.jpg" title="Set the connection as Encrypted" />
+6. Click **Create**. This will run a connection test (unless you choose to skip it). If the connection succeeds, the connection will be saved.
+
+You can now go back to your published report on Power BI Service to assert data loads as expected.
+
+## Use the connector
+
+This section describes how to use the dbt Semantic Layer connector in Power BI.
+
+The dbt Semantic Layer connector creates:
+- A virtual table for each saved query.
+- A `METRICS.ALL` table containing all metrics, and dimensions and entities appear as regular dimension columns.
+
+These tables do not actually map to an underlying table in your data warehouse. Instead, Power BI sends queries to these tables and (before actually executing on the warehouse) the Semantic Layer servers:
+- Parse the SQL.
+- Extract all the queried columns, group bys and filters.
+- Generates SQL to query your existing tables. 
+- Returns data back to Power BI, which doesn’t know any of this happened.
+
+<Lightbox src="/img/docs/cloud-integrations/sl-pbi/sl-pbi.jpg" width="90%" title="Power BI integration diagram" />
+
+This allows for very flexible analytics workflows, like drag and drop metrics and slice by dimensions and entities &mdash; the Semantic Layer will generate the appropriate SQL to actually query your data source for you.
+
+## Considerations
```

Comment:
We'll want to add another consideration:

- Custom modeling is not supported
  - Joining tables is not supported
  - Creating custom columns within a table is not supported
  - No custom DAX or PQ is supported (although some of it might work)